scripts/version_check.py

```diff
@@ -118,15 +128,15 @@ def _main_debug(args):
     return _SUCCESS
 
 
-def _main(args):
+def _main(args: Namespace) -> int:
     try:
         return _main_debug(args)
     except Exception as ex:
         print(f"Error while executing program, details: {ex}", file=sys.stderr)
         return _FAILURE
 
 
-def main(argv=None):
+def main(argv: Union[Iterable[str], None] = None) -> int:
```

Comment:
uff

Reply:
yep, but typing in concrete type's is a bad habit imho ;p